local-cli/rnpm/wpf/package.json

```diff
@@ -24,8 +21,6 @@
     "react-native": ">=0.31.0"
   },
   "dependencies": {
-    "semver": "^5.0.3",
-    "chalk": "^1.1.3",
-    "node-fetch": "^1.3.3"
+    "rnpm-plugin-windows": "^0.2.9"
   }
```

Comment:
This dependency range is actually >=0.2.9 <0.3.0. Should we just set the range to >=0.2.9?